doc/user_guide/index.rst

```diff
@@ -48,27 +48,29 @@ PSyclone User Guide
 .. only:: html
 
     PSyclone is a source-to-source Fortran compiler designed to programmatically
-    optimise and parallelise HPC applications using user-provided transformation
-    scripts. These scripts offer a separation of concerns between the science code
-    and the performance portability implementation choices, allowing to experiment
-    and develop with each side independenly.
-    Additonally, PSyclone supports the development of kernel-based Fortran-embedded
-    DSLs following the PSyKAl model
-    developed in the
+    optimise, parallelise and instrument HPC applications via user-provide
+    transformation scripts.
+
+    By encapsulating the performance-portability aspects (e.g. whether to
+    parallelise with OpenMP or OpenACC), these scripts enable a separation of
+    concerns between the scientific implementation and the optimisation choices.
+    This allows each aspect to be explored and developed largely independently.
+    Additionally, PSyclone supports the development of kernel-based Fortran-embedded
+    DSLs following the PSyKAl model developed in the
     `GungHo project <https://www.metoffice.gov.uk/research/foundation/dynamics/next-generation>`_.
 
-    Notably, it is used to support the
-    `LFRic <https://www.metoffice.gov.uk/research/modelling-systems/lfric/>`
-    PSyKAl mixed finite-element DSL for the UK MetOffice's next generation
+    PSyclone is currently used to support the
+    `LFRic <https://www.metoffice.gov.uk/research/modelling-systems/lfric/>`_
+    mixed finite-element PSyKAl DSL for the UK MetOffice's next generation
     modelling system and the
     `GOcean <https://gtr.ukri.org/projects?ref=NE%2FL01209X%2F1>`_
-    finite difference prototype DSL.
+    finite-difference PSyKAl DSL for a prototype 2D ocean modelling system.
     It is also used to insert GPU offloading directives to existing
     directly-addressed MPI applications such as the
     `NEMO ocean model <https://www.nemo-ocean.eu/>`_ ocean model.
```

Comment:
"ocean model" is repeated so I suggest: s/ocean model././

Reply:
Done